phys/module_sf_bep_bem.F

```diff
@@ -1462,177 +1706,39 @@ subroutine BEP1D(iurb,kms,kme,kts,kte,z,dt,ua,va,pt,da,pr,pt0,   &
              enddo
           enddo
-       
-         !print*,'nz_um',nz_um
-         !print*,'nlev(ibui)',nlev(ibui)
-         !print*,'nhourday',nhourday
-         !print*,'dt',dt
-         !print*, 'bs_u(1,iurb)',bs_u(1,iurb)
-         !print*, 'bs_u(2,iurb)',bs_u(2,iurb)
-         !print*, 'dz_u',dz_u
-         !print*, 'nwr_u',nwr_u
-         !print*, 'nf_u',nf_u
-         !print*, 'nwr_u', nwr_u
-         !print*, 'ngb_u',ngb_u
-         !print*, 'sfwb1D',sfwb1D
-         !print*, 'gfwb1D',gfwb1D
-         !print*, 'sfwinb1D',sfwinb1D
-         !print*, 'sfrb(1,ibui)',sfrb(1,ibui)
-         !print*, 'gfrb(1,ibui)',gfrb(1,ibui)
-         !print*, 'latent',latent
-         !print*,  'sigma',sigma
-         !print*, 'albw_u(iurb)',albw
-         !print*, 'albwin_u(iurb)',albwin
-         !print*, 'albr_u(iurb)',albr
-         !print*, 'emr_u(iurb)',emr
-         !print*, 'emw_u(iurb)',emw
-         !print*, 'emwind_u(iurb)',emwind
-         !print*, 'rsw',rsw
-         !print*, 'rlw',rlw
-         !print*, 'r',r
-         !print*, 'cp_u',cp_u
-         !print*, 'da_u',da_u
-         !print*, 'tmp_u',tmp_u
-         !print*, 'qv_u',qv_u
-         !print*, 'pr_u',pr_u
-         !print*, 'rs',rs
-         !print*, 'rld',rld
-         !print*, 'dzw',dzw
-         !print*, 'csw',csw
-         !print*, 'alaw',alaw
-         !print*, 'pwin_u',pwin_u
-         !print*, 'cop_u(iurb)',cop_u(iurb)
-         !print*, 'beta_u(iurb)',beta_u(iurb)
-         !print*, 'sw_cond_u(iurb)',sw_cond_u(iurb)
-         !print*, 'time_on_u(iurb)',time_on_u(iurb)
-         !print*, 'time_off_u(iurb)',time_off_u(iurb)
-         !print*, 'targtemp_u(iurb)',targtemp_u(iurb)
-         !print*, 'gaptemp_u(iurb)',gaptemp_u(iurb)
-         !print*, 'targhum_u(iurb)',targhum_u(iurb)
-         !print*, 'gaphum_u(iurb)',gaphum_u(iurb)
-         !print*, 'perflo_u(iurb)',perflo_u(iurb)
-         !print*, 'hsesf_u(iurb)',hsesf_u(iurb)
-         !print*, 'hsequip',hsequip
-         !print*, 'dzf',dzf
-         !print*, 'csf',csf
-         !print*, 'alaf',alaf
-         !print*, 'dzgb',dzgb
-         !print*, 'csgb',csgb
-         !print*, 'alagb',alagb
-         !print*, 'dzr',dzr
-         !print*, 'csr',csr
-         !print*, 'alar',alar
-         !print*, 'tlevb1D',tlevb1D
-         !print*, 'qlevb1D',qlevb1D
-         !print*, 'twb1D',twb1D
-         !print*, 'twlevb1D',twlevb1D
-         !print*, 'tflevb1D',tflevb1D
-         !print*, 'tglevb1D',tglevb1D
-         !print*, 'trb1D',trb1D
-         !print*, 'sflev1D',sflev1D
-         !print*, 'lflev1D',lflev1D
-         !print*, 'consumlev1D',consumlev1D
-         !print*, 'sfvlev1D',sfvlev1D
-         !print*, 'lfvlev1D',lfvlev1D
-
 
+    !print*,'HFGR_BEFORE_CALLING_BEM',hfgr(nlev(ibui))
 
-
-     
-         
-          call BEM(nz_um,nlev(ibui),nhourday,dt,bs_u(1,iurb),                &
+                      call BEM(nz_um,nlev(ibui),nhourday,dt,bs_u(1,iurb),                &
```

Comment:
more indented than before

Reply:
Jimy,
Agreed. I lined up the "call" with the previous indenting.